include/sched.h

```diff
@@ -252,9 +252,10 @@ int    sched_setaffinity(pid_t pid, size_t cpusetsize,
                          FAR const cpu_set_t *mask);
 int    sched_getaffinity(pid_t pid, size_t cpusetsize, FAR cpu_set_t *mask);
 int    sched_cpucount(FAR const cpu_set_t *set);
-int    sched_getcpu(void);
 #endif /* CONFIG_SMP */
 
+int    sched_getcpu(void);
```

Comment:
```suggestion
#else
#  define sched_getcpu() 0
#endif /* CONFIG_SMP */
```
I mean in this way and keep the rest of the code unchanged

Reply:
Ok, changed.